cypress/e2e/right-panel/right-panel.spec.ts

```diff
@@ -58,6 +66,31 @@ describe("RightPanel", () => {
     });
 
     describe("in rooms", () => {
+        it("should handle long room address and long room name", () => {
+            cy.createRoom({ name: ROOM_NAME_LONG });
+            viewRoomSummaryByName(ROOM_NAME_LONG);
+
+            cy.openRoomSettings();
+
+            // Set a local room address
+            cy.findByTestId("local-address-fieldset").within(() => {
+                cy.get(".mx_AliasSettings_localAddresses").click();
+                cy.findByLabelText("Room address").type(ROOM_ADDRESS_LONG);
+                cy.findByRole("button", { name: "Add" }).click();
+            });
+
+            cy.closeDialog();
+
+            // Close and reopen the right panel to render the room address
+            cy.findByRole("button", { name: "Room info" }).click();
+            cy.get(".mx_RightPanel").should("not.exist");
+            cy.findByRole("button", { name: "Room info" }).click();
+
+            cy.get(".mx_RightPanel").percySnapshotElement("RoomSummaryCard - with a room name and a local address", {
+                widths: [264], // Emulate the UI. The value is based on minWidth specified on MainSplit.tsx
```

Comment:
The value is available here: https://github.com/matrix-org/matrix-react-sdk/blob/de16d347e9294f9f4771ca4bc950fcf1e8c1acb4/src/components/structures/MainSplit.tsx#L74